runtime/wasm/src/to_from/external.rs

```diff
@@ -58,6 +58,24 @@ impl FromAscObj<AscBigInt> for BigInt {
     }
 }
 
+impl ToAscObj<AscBigDecimal> for BigDecimal {
+    fn to_asc_obj<H: AscHeap>(&self, heap: &mut H) -> AscBigDecimal {
+        // "exponent" here is the opposite of what you'd expect.
```

Comment:
Could we make this a little more specific? I found `Note that a positive exponent indicates a negative power of 10` in the `BigDecimal` docs, which made it more clear: https://docs.diesel.rs/bigdecimal/struct.BigDecimal.html#method.as_bigint_and_exponent

Reply:
On another note: Is this a normal thing to do, do we know? It's so weird. 😉

Reply:
Don't know why the Rust library does that, it's also weird to me. I checked the Python library and it uses exponents.